### PR TITLE
Add current status date between scope

### PR DESCRIPTION
**Pull request by ahmad-shawky**
Add current status date between scope to get models with latest status date between dates

### CODE REVIEW

**Comment by freekmurze** [comment]
Thanks for your work on this.

There are some conflicts, could you solve those.

Could you also add an example to the readme with an explanation what the scope does exactly?

**Comment by ahmad-shawky** [comment]
@freekmurze Thank you very much. I resolved the conflicts and updated readme.

File: README.md [hunk 1]
```diff
@@ -170,6 +170,16 @@ $allNonInitiatedOrPendingModels = Model::otherCurrentStatus(['initiated', 'pendi
 $allNonInitiatedOrPendingModels = Model::otherCurrentStatus('initiated', 'pending');
 ```
 
+### Retrieving models that have current status created between two dates
```
Comment (freekmurze) [inline README.md]: What do you mean with "current status". Do you mean any status?

Comment (ahmad-shawky) [reply README.md]: I mean by current status latest status.

**Comment by freekmurze** [comment]
After thinking it through, I've decided not to merge this PR because I want to keep it light and don't have a need or want to maintain the introduced code.

Thanks for your work on it though.

**Pull request closed without merging**